examples/hybrid/sphere/single_column_radiative_equilibrium.jl

```diff
@@ -12,17 +12,19 @@ params = EarthParameterSet()
 horizontal_mesh =
     periodic_rectangle_mesh(; x_max = Δx, y_max = Δx, x_elem = 1, y_elem = 1)
 quad = Spaces.Quadratures.GL{1}()
-z_max = FT(100e3)
-z_elem = 100
-t_end = FT(60 * 60 * 24 * 365.25 * 5)
+z_max = FT(90e3)
+z_elem = 90
+z_stretch = Meshes.GeneralizedExponentialStretching(FT(100), FT(10000))
+t_end = FT(60 * 60 * 24 * 365.25)
 dt = FT(60 * 60 * 3)
-dt_save_to_sol = 100 * dt
+dt_save_to_sol = 10 * dt
```

Comment:
Can we use the cli option here instead?

Reply:
Same as with the other comment---I'd like to set a default value for this example, but have that default be different from the one for the baroclinic wave examples (1 day). Right now, this value is overriding the CL value, and that order of precedence should definitely be reversed, but I'll fix that when I modify the driver for REPL usage.